lib/player.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "./scoring.rb"
+require_relative "./tilebag.rb"
+
+module Scrabble
+  class Player < TileBag
+
+    attr_reader :name
+
+    attr_accessor :won, :plays, :tiles
+
+    def initialize(name)
+      @name = name
+      @plays = []
+      @tiles = []
+    end
+
+    def play(word)
+      return false if won?
+      @plays << word
+      Scoring.score(word)
+    end
+
+    def total_score
+      scores = @plays.map do |word|
+        Scoring.score(word)
+      end
+      return scores.sum
+    end
+
+    def won?
+      return total_score > 100
+    end
+
+    def highest_scoring_word
+      Scoring.highest_score_from
+    end
```

Comment:
Doesn't `highest_score_from` take an argument? Looks like you don't have any tests for this method, otherwise they would definitely have gotten an `ArgumentError`.